docs/classic-vs-vt.md

```diff
@@ -0,0 +1,81 @@
+
+While the Windows way of handling this responsibility in the console host and API makes it quicker and easier to write a command-line application with these features and without responsibility of remembering drawing state or handling editing convenience features, it makes it nearly impossible to connect those activities remotely across platforms, versions, or scenarios due to variations in implementations and availability. It also makes the final interactive experience of these Windows command-line applications completely dependent on the console host's implementation, priorities, and release cycle.
+
+By contrast, other platforms handle these activities and virtual terminal communication itself through reusable client-side libraries like [readline](https://tiswww.case.edu/php/chet/readline/rltop.html) and [ncurses](https://invisible-island.net/ncurses/ncurses.html). The final terminal is only responsible for displaying information and receiving input through that bidirectional communication channel.
+
+### Wrong-Way Verbs
+
+On Windows, some actions can be performed in the opposite-to-natural direction on the input and output streams. This allows Windows command-line applications to both avoid the concern of managing their own buffers and to perform advanced operations like simulating/injecting input on behalf of a user or reading back some of the history of what was written. While this provides additional power to Windows applications operating in a specific user context on a single machine, it also provides a vector to cross security and privilege levels or domains when used in scenarios between contexts on the same machine or even across contexts to another machine or environment. Other platforms do not allow this activity and our intent is to converge with that strategy for both interoperability and security reasons.
+
+### Direct Window Access
+
+On Windows, the exact window handle to the hosting window is available through the Console API surface. This allows a command-line utility to perform advanced window operations by reaching into the wide gamut of Win32 APIs permitted against a window handle and manipulate the window state, frame, icon, or other properties about the window. By contrast, on other platforms with virtual terminal sequences, there is a specific gamut of commands that can be performed against the window to do things like changing its size or displayed title and they must be done in the same band and under the same control as the remainder of the stream.

```

Comment:
`specific gamut` -> `narrow set` -- using `gamut` 2x in same paragraph stuck out to me